src/main/java/bio/terra/app/model/GoogleCloudResource.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.app.model;
+
+/**
+ * Google cloud resources used by TDR.
+ */
+public enum GoogleCloudResource {
+    BIGQUERY("bigquery"), FIRESTORE("firestore"), BUCKET("bucket");
+
+    private String value;
+
+    GoogleCloudResource(String value) {
+        this.value = value;
+    }
+
+    public String toString() {
+        return String.valueOf(value);
+    }
+
+    public static GoogleCloudResource fromValue(String text) {
+        for (GoogleCloudResource b : GoogleCloudResource.values()) {
+            if (String.valueOf(b.value).equals(text)) {
```

Comment:
Redundant `String.valueOf()` (since `b.value` is `String`)
```suggestion
            if (b.value.equals(text)) {
```

Reply:
ugh this was autogenerated by swagger and I just moved it into the repo. Swagger autogen code really is kinda ugly...

Reply:
That explains a lot!